keadm/cmd/keadm/app/cmd/util/image.go

```diff
@@ -281,8 +281,6 @@ func (runtime *CRIRuntime) CopyResources(edgeImage string, files map[string]stri
 			SecurityContext: &runtimeapi.LinuxSandboxSecurityContext{
 				NamespaceOptions: &runtimeapi.NamespaceOption{
 					Network: runtimeapi.NamespaceMode_POD,
-					Pid:     runtimeapi.NamespaceMode_CONTAINER,
-					Ipc:     runtimeapi.NamespaceMode_POD,
```

Comment:
Plz give more discriptions for why delete `pid` and `ipc` above :)

Reply:
> Plz give more discriptions for why delete `pid` and `ipc` above :)

Using the default value for Pid and Ipc is enough for copy-resource container. If we set Pid with NamespaceMode_CONTAINER, keadm join with cri-o runtime will fail to create copy-resource container like #5139

Reply:
Okey, sounds good, but the `Pid` and `Ipc` above all use the same default values with k8s, not sure what's the difference with k8s.. cc @Shelley-BaoYue @WillardHu